mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -30,6 +31,8 @@
 DIRECTIONS = ['-F 1', '-F 2', '-F 4']
 DATA_TYPES = ['conv', 'convfp16', 'convint8']
 LAYOUTS = ['NHWC', 'NCHW']
+# Compiled regexp object used for extracting elapsed time from MIOpenDriver's output
+pattern = re.compile(r"Elapsed: (.*)ms")
```

Comment:
This sort of thing should really be an `ALL_CAPS_CONSTANT` - perhaps `ELAPSED_TIME_RE`